include/Dialect/TfheRust/IR/TfheRustOps.td

```diff
@@ -43,5 +44,16 @@ def ApplyLookupTableOp : TfheRust_Op<"apply_lookup_table"> {
   let results = (outs TfheRust_CiphertextType:$output);
 }
 
+def GenerateLookupTableOp : TfheRust_Op<"generate_lookup_table"> {
+  let arguments = (
+    ins TfheRust_ServerKey:$serverKey,
+    // TODO(https://github.com/google/heir/issues/246): Generalize to support
+    // integer-valued lookup tables; for now the integer represents a
+    // binary-valued truth table as a bit string, evaluated via
+    // `(lut >> input) & 1`
```

Comment:
Is this for the future world where we use the shortint API and not-booleanize?

Reply:
Yup!